README.md

```diff
@@ -35,27 +35,24 @@ Atlas will serve as a bridge from existing bare-metal HPC clusters (example: Sat
+- **Volumes:** Used for save training/test/validation data of AI program as well as results of the program.
 
-> `Figure 1` and `Figure 2` below show the overview architecture of this project and the rough sequence among each component.
+> `Figure 1` below shows the overview architecture of this project.
 
-![alt text](./imgs/../doc/imgs/overview-architecture.png "Overview Architecture")
+![alt text](https://www.lucidchart.com/publicSegments/view/4f14108f-deb4-4e67-836f-cd1049b2fe00/image.png "Overview Architecture")
 <p align="center">Figure 1: Global Architectural Structure Of the Project</p>
 
-![alt text](./imgs/../doc/imgs/overview-sequence-diagram.png "Overview Sequence Diagram")
-<p align="center">Figure 2: Sequence diagram of this project</p>
+<!-- ![alt text](./imgs/../doc/imgs/overview-sequence-diagram.png "Overview Sequence Diagram")
+<p align="center">Figure 2: Sequence diagram of this project</p> -->
 
 ### Design Implications and Discussion:
 This section discusses the implications and reasons for the design decisions made during the global architecture design.
 - **Scripts/Executables:**
-In order to compare two systems benefits, the Scripts/Executables will be needed to easily upload the codes to the bare-metal system and cloud-native (OpenShift) at the same time. And the scripts/executables will be one of the most important parts of the whole workflow since it not only sends the codes but also sends results from the bare-metal system to the OpenShift database to let task monitor service compare and then return the comparison results to the users.
-- **Task monitor service:** 
-Since the tasks need uncertain time to complete, and may fail at any time, task monitor service will be needed to keep track of the detailed status of tasks, to see if each step finishes or fails (data preprocessing, data training, data prediction and so on), then after all tasks finishing, it will gather all the results to form the final deliverables.
-- **Outside data import interface:**
-Besides the data directly from users, systems will be able to retrieve the data from an external resource and able to do data screening. Because normally, the size of those data is very large, typically in gigabytes. 
+In order to compare two systems benefits, the Scripts/Executables will be needed to easily upload the codes to the bare-metal system and cloud-native (OpenShift) at the same time. And the scripts/executables will be one of the most important parts of the whole workflow since it will tell how OpenShift and Satori do to make the AI workflow work, and get the returned comparison results to the users.
+- **Containers:** On OpenShift, we should use **BuildConfig** to automatically build images for AI workflows and deploy them as containers. Each container can only serve for one application.
+- **Volumes:** For persistent storage, save data on containers or retrieve from internet are not a wise choice, so we decide to use volume on OpenShift to store our data.
```

Comment:
> save data on containers or retrieve from internet are not a wise choice

Suggestion: save data in containers or downloading every single time is not a wise choice